client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/step-content.tsx

```diff
@@ -102,6 +102,32 @@ jest.mock( '@automattic/data-stores', () => ( {
 					},
 				];
 				break;
+
+			case 'design-first':
+				checklist = [
+					{ id: 'select_design', completed: false, disabled: false, title: 'Select a design' },
```

Comment:
```suggestion
					{ id: 'design_selected', completed: false, disabled: false, title: 'Select a design' },
```
I think this is the correct id is according to the response I got from `/launchpad?checklist_slug=design-first`

Reply:
Good catch, [changed](https://github.com/Automattic/wp-calypso/pull/77222/commits/fb3ce06b5fffb5d2cf9c181171c220dc5ac26719).